resources/views/pages/design.twig

```diff
@@ -159,6 +159,68 @@
                 </p>
             </div>
         </div>
+            <div class="col-6 col-md-12 mb-4">
+            <span id="icons" class="ref-id"></span>
+            <h3>Icons <a href="#icons" class="ref-link">{{ m.icon('link') }}</a></h3>
+            <div class="row">
+                <div class="col-md-6 col-lg-4 mb-4">
+                    cancel: {{ m.icon('x-lg') }}<br>
+                    delete: {{ m.icon('trash', 'danger') }}<br>
+                    edit: {{ m.icon('pencil') }}<br>
+                    preview: {{ m.icon('eye','info') }}<br>
+                    reset: {{ m.icon('arrow-counterclockwise') }}<br>
+                    save: {{ m.icon('save', 'primary') }}<br>
+                    <br>
+                    {{ f.button('Cancel', {'icon_left': 'x-lg'}) }}<br>
```

Comment:
You could add a `class="mb-3"` to the br so that it has better spacing

Reply:
wont do